src/manipulation.js

```diff
@@ -0,0 +1,80 @@
+import Todo from './todo';
+
+// Display list
+const display = () => {
+  const task = new Todo();
+  const localTasks = JSON.parse(localStorage.getItem('localTasks'));
+  const items = document.querySelector('.items');
+
+  if (localTasks != null) {
+    localTasks.forEach((localtask) => {
+      const tt = {
+        desc: localtask.desc,
+        completed: localtask.completed,
+        index: localtask.index,
+      };
+      task.tasks.push(tt);
+      if (localtask.completed === true) {
+        items.innerHTML += `<form class='item item${localtask.index}'>
+      <div class='item-desc'><div class='check'><input type='checkbox' id='check${localtask.index}' class='checkbox checkbox${localtask.index}' checked></div><input type='text' id='desc${localtask.index}' class='desc completed desc${localtask.index}' value='${localtask.desc}'></div>
+      <div class='humberger'><img class='humberger-image humberger-image${localtask.index}' src='assets/images/more.png' alt='humberger'> <img class='trash-image trash-image${localtask.index}' src='assets/images/trash.png' alt='trash'></div>
+      </form>`;
+      } else {
+        items.innerHTML += `<form class='item item${localtask.index}'>
+      <div class='item-desc'><div class='check'><input type='checkbox' id='check${localtask.index}' class='checkbox checkbox${localtask.index}'></div><input type='text' id='desc${localtask.index}' class='desc desc${localtask.index}' value='${localtask.desc}'></div>
+      <div class='humberger'><img class='humberger-image humberger-image${localtask.index}' src='assets/images/more.png' alt='humberger'> <img class='trash-image trash-image${localtask.index}' src='assets/images/trash.png' alt='trash'></div>
+      </form>`;
```

Comment:
Line18 is repeating itself in 23, lines 20-21 are repeating themselves in line 25-26. Kindly write them out of the conditional block to follow the DRY practice.